docs/docs/10.5-clone-with-props.md

```diff
@@ -1,12 +1,12 @@
 ---
 id: clone-with-props
-title: Cloning ReactElement
+title: Cloning ReactElements
 permalink: clone-with-props.html
 prev: test-utils.html
 next: create-fragment.html
 ---
 
-In rare situations a element may want to change the props of a element that it doesn't own (like changing the `className` of a element passed as `this.props.children`). Other times it may want to make multiple copies of a element passed to it. `cloneWithProps()` makes this possible.
+You can use `cloneWithProps()` to copy an element and optionally set `props` on the copies. For example, you can set the `className`s of a elements accessible via `this.props.children`.
```

Comment:
Hi jsfb

Thanks for the feedback. I'll incorporate the changes. Could you recommend a title? The current title is "Cloning ReactElement," which is confusing since `React.cloneElement` is similar but distinct. 

I also think it would be best if the documentation could provide guidance as to the use case for using `React.addons.cloneWithProps` instead of `React.cloneElement`. Here's my guess: `cloneWithProps` is useful for when you want to create multiple copies of an element rather than a single copy. That's when you would definitely *not* want to keep `key` and `ref`. Is that right?